peachjam/tasks.py

```diff
@@ -250,7 +250,32 @@ def update_user_follows_for_user(user_id):
         return
 
     log.info(f"Updating user follows for user {user_id}")
-    UserFollowing.update_and_alert(user)
+    UserFollowing.update_timeline(user)
+
+
+@background(queue="peachjam", remove_existing_tasks=True, schedule={"priority": -1})
```

Comment:
i've set the email tasks at a lower priority so that they run after the update follow tasks.